src/tutorial/index.rst

```diff
@@ -435,30 +421,40 @@ Below you can see how the message output function may be assembled. Normally, ag
     }
     """
     ...
+
+  .. code-tab:: py Agent Python
+
+    ...
+    # Agent Function to output the agents ID and position in to a 2D spatial message list
+    @pyflamegpu.agent_function
+    def output_message(message_in: pyflamegpu.MessageNone, message_out: pyflamegpu.MessageSpatial2D):
+        message_out.setVariableUInt("id", pyflamegpu.getID())
+        message_out.setLocation( \
```

Comment:
Are these slashes actually required?

Reply:
Actually no and Pep suggests not to so removed.